### PR TITLE
Configurable precision

### PR DESCRIPTION
**Pull request by cpennington**
Use `--precision X` to store `X` digits of response_times.

### CODE REVIEW

**Comment by cpennington** [comment]
@jimabramson @BenjiLee @cgoldberg @doctoryes: Review?

**Comment by feanil** [comment]
@cpennington is there a plan to also PR this back upstream?

**Comment by cpennington** [comment]
Already is one: https://github.com/locustio/locust/pull/258

File: locust/stats.py [hunk 1]
```diff
@@ -157,22 +169,25 @@ def _log_response_time(self, response_time):
...
+        rounded_response_time = self._round_response_time(response_time)
 
         # increase request count for the rounded key in response time dict
         self.response_times.setdefault(rounded_response_time, 0)
         self.response_times[rounded_response_time] += 1
 
+    def _round_response_time(self, response_time):
+        # to avoid to much data that has to be transfered to the master node when
+        # running in distributed mode, we save the response time rounded in a dict
+        # so that 147 becomes 150, 3432 becomes 3400 and 58760 becomes 59000
```
Comment (jimabramson) [inline locust/stats.py]: i think this comment should be updated to reflect the added configurability.

**Comment by BenjiLee** [comment]
Didn't realize there were two PRs for this. :+1:

**Comment by jimabramson** [comment]
other than my request for updated comment, :+1:

**Pull request merged**